src/main/java/seedu/mtracker/model/InstrumentManager.java

```diff
@@ -44,10 +44,6 @@ public void addInstrument(Instrument addedInstrument) {
         instruments.add(addedInstrument);
     }
 
-    public void doneInstrument(int completedInstrumentIndex) {
-        instruments.get(completedInstrumentIndex).markAsDone();
-    }
-

```

Comment:
Can test the program see whether done still works, I am not sure if you initialise an instrument in the done command and set that instrument as done whether it actually set the instrument in the manager as done also. TLDR, I am not sure if they are 2 different objects. If it works sure lets continue with the implementation

Reply:
Yes it works. getInstrumentAtIndex() method in IndexedCommand class gets the instrument at the index insider InstrumentManager's instruments. Then we are just updating the done status of that instrument at that index of instruments arraylist.